tests/console/sle15_workarounds.pm

```diff
@@ -15,15 +15,21 @@
 # Summary: performing extra actions specific to sle 15 which are not available normally
 # Maintainer: Rodion Iafarov <riafarov@suse.com>
 
-use base "consoletest";
+use base qw(consoletest distribution);
 use strict;
 use testapi;
 use utils qw(zypper_call sle_version_at_least);
 
 
 sub run {
+    my $self = shift;
     return unless sle_version_at_least('15');
-    select_console 'root-console';
+    send_key('ctrl-alt-f2');
+    #assert_screen(["tty$tty-selected", 'text-login', 'text-logged-in-root', 'generic-desktop']);
+    if (check_screen 'generic-desktop', 0) {
```

Comment:
rather should be assert_screen+match_has_tag or wait_still_screen but not dead code